vendors/infineon/boards/xmc4800_iotkit/aws_tests/application_code/main.c

```diff
@@ -93,8 +93,12 @@ int main( void )
                             tskIDLE_PRIORITY,
                             mainLOGGING_MESSAGE_QUEUE_LENGTH );
 
+    /* Initialize AWS Libraries. */
+    SYSTEM_Init();
```

Comment:
Per the comment a few lines below, this was previously being done after the scheduler was started (albeit within the daemon task callback). Does this have any dependencies on that?